docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,148 @@
+
+```shell script
+kubectl logs api-gateway-api-migrator-job -n kyma-system
+```
+
+You will find logs saying which Api was not migrated and a reason for that.
+
+If the Api was not migrated due to the invalid status or a blacklisted label, you either shouldn't or will not be able to do the migration.
+
+## Manual migration
+
+The manual migration process consists of the following steps:
+
+1. Get Api resource you want to migrate. It would be helpful to have it open during the whole migration process.
+
+```shell script
+kubectl get api <NAME> -n <NAMESPACE>
+```
+
+2. Create APIRule resource with a different host
+
+Here is the documentation for both custom resources, which may be useful during the migration:
+
+https://kyma-project.io/docs/1.11/components/api-gateway/#custom-resource-api-sample-custom-resource
+https://kyma-project.io/docs/1.11/components/api-gateway-v2#custom-resource-api-rule
+
+Only the `spec` part of the CRD will be described. `metadata` field can be adjusted in any way and `status` field should not be copied.
+
+Starting from the `service` field of APIRule, copy `name` and `port` fields from the `service` field of Api object. Please set `host` value to any temporary value including the domain. For example, if the `hostname` value of Api was set to `sample-service.kyma.local` you can change it to `temp-sample-service.kyma.local`. Just make sure that the hostname is not used by other services on your cluster. That value will be changed in a later step, but for now it needs to be different than the original value. Set `gateway` field of APIRule to the Istio Gateway used to expose services. By default it will be `kyma-gateway.kyma-system.svc.cluster.local`.
+
+The configuration of the `rules` field of APIRule is more complex and depends on the `authentication` configuration of APIRule. As all the basic scenarios are covered by automatic migration, this explanation will only concern the configurations that are not handled automatically.
+
+The basic difference between Apis and APIRules authentication configuration is that while Api allows to enable the authentication for the whole service and disable it on specific paths only, the APIRule has an approach where you specify what authentication should be used per specific path (including the possibility to set it for all paths) and the paths must not cross each other. Another important difference is that Api supports a list of issuers and jwks URIs, but excluded paths are set independently on both. It means that in the example below to access `/exact/path/to/resource.jpg` path the token issued from `https://auth.kyma.local` is required, to access any path starting with `/pref/` the token issued from `https://dex.kyma.local` is required, to access the `/no/auth/needed/resource.html` no token is required because it is excluded for both settings and to access all other paths the token from one of the issuers is required.
```

Comment:
> ... and the paths must not cross each other.

Perhaps "must not overlap" would be more accurate? You could also provide an example of different yet overlapping paths.

Reply:
> It means that in the example below to access `/exact/path/to/resource.jpg` path the token issued from `https://auth.kyma.local` is required, to access any path starting with `/pref/` the token issued from `https://dex.kyma.local` is required, to access the `/no/auth/needed/resource.html` no token is required because it is excluded for both settings and to access all other paths the token from one of the issuers is required.

Again, bullet points would make this section transparent. - Path, authentication method (if any), commentary.

Reply:
I prefer more strict approach: "must not overlap" -> "must be unique"

Reply:
IMHO "must be unique" is not sufficient here. You can say that `/.*` and `/something/.*` are two unique paths, but you can't use that configuration, as they overlap.